app/content_script.css

```diff
@@ -1,4 +1,4 @@
-#interceptor-container {
+.interceptor-container {
   position: absolute;
   top: 10px;
   right: 10px;
```

Comment:
Since content script is removed, we might as well remove everything used by the content script.